src/main/scala/assembler/AssemblerMain.scala

```diff
@@ -0,0 +1,18 @@
+package assembler
+
+
+import global.Configs._
+import service.EnterpriseAssemblerService
+
+
+object AssemblerMain extends EnterpriseAssemblerService {
+
+  def main(args: Array[String]) {
+
+     updateConf(args)
+     loadFromParquet /*loadFromJson*/
```

Comment:
Maybe use an environment variable as a condition on whether to load from Parquet or JSON